.github/workflows/psalm.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         dependencies:
-          - "locked"
+          - "highest"
```

Comment:
`vimeo/psalm` constantly changes static analysis checks as type inference improves: we need to run it with locked dependencies, or else build becomes too unstable

Reply:
> 
> 
> please look at https://github.com/Roave/no-leaks/pull/124/checks?check_run_id=2853673697 failing test, it is failing randomly and it seems unrelated with this PR

did you read it?